### PR TITLE
Dependency injection partial application

### PR DESCRIPTION
**Pull request by hyeomans**
I added `_drafts` as part of the gitignore file. That way we can work locally in any ideas we have and just open PR's for actual posts.

@keogh

### CODE REVIEW

File: source/_posts/selling-your-solid-oop-principles.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Selling your SOLID OOP principles
+date: 2018-05-23
+author: Hector Yeomans
+draft: true
```
Comment (keogh) [inline source/_posts/selling-your-solid-oop-principles.md]: I think this is not the right way to do a draft, you need to `hexo new draft title`

Comment (hyeomans) [reply source/_posts/selling-your-solid-oop-principles.md]: Ah got it, I don't remember adding this.

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,6 @@ db.json
 *.log
 node_modules/
 public/
-.deploy*/
\ No newline at end of file
+.deploy*/
+
+source/_drafts
```
Comment (keogh) [inline .gitignore]: I think this should be in the source control

Comment (hyeomans) [reply .gitignore]: Why? I see tradeoffs on either including them or removing it.

If we include them, then our PR's will have data that we don't want to be reviewed.
If you exclude them, then we loose our ability to work on different computers given that they will be ignored.

Comment (keogh) [reply .gitignore]: What worries is to repeat the mistake I did, I accidentally removed a post and it wasn't in the source control u.u

Maybe we should not add draft posts to the PRs?

Comment (hyeomans) [reply .gitignore]: Yeah, we can keep _drafts in source control and don't add them to PR's if we see is a PITA to keep track we can later remove it.

**Review by keogh — APPROVED** [review]

**Pull request merged**